client/src/helpers/UserHelpers.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
```

Comment:
We can also use an axios config to do some of this work. Maybe we chat about somehow fusing this and my api in the profile page?

Reply:
That would be great! And I'd need some help understanding it as well

Reply:
Yes that's be cool

Reply:
I'll think about it and bring it up at standup

Reply:
Decided we will do this in @tsyaeger PR